ts/store/actions/persistedPreferences.ts

```diff
@@ -17,9 +17,9 @@ export const serviceAlertDisplayedOnceSuccess = createStandardAction(
   "SERVICE_ALERT_DISPLAYED_ONCE_SUCCESS"
 )<{ wasServiceAlertDisplayedOnce: boolean }>();
 
-export const pagoPaQAEnvironmentEnabledSuccess = createStandardAction(
-  "PAGOPA_ENV_AS_QA"
-)<{ isPagoPAQAEnabled: boolean }>();
+export const pagoPaTestEnvironmentEnabledSuccess = createStandardAction(
```

Comment:
follow the format of `preferencesExperimentalFeaturesSetEnabled`: `preferencesPagoPATestEnvironmentSetEnabled`